lightning/src/chain/channelmonitor.rs

```diff
@@ -332,14 +332,15 @@ impl Readable for CounterpartyCommitmentParameters {
 	}
 }
 
-/// An entry for an [`OnchainEvent`], stating the block height when the event was observed and the
-/// transaction causing it.
+/// An entry for an [`OnchainEvent`], stating the block height and hash when the event was
+/// observed, as well as the transaction causing it.
 ///
 /// Used to determine when the on-chain event can be considered safe from a chain reorganization.
 #[derive(PartialEq, Eq)]
 struct OnchainEventEntry {
 	txid: Txid,
 	height: u32,
+	block_hash: Option<BlockHash>,
```

Comment:
Could add a similar comment as `transaction` to know when this was added.

Reply:
Done.